Add color token for error background

misolori opened the pull request:
Fixes #16204 and adds a new color token for error background colors, `editorError.background`:

![image](https://user-images.githubusercontent.com/35271042/98416715-71871b00-2034-11eb-9da2-9910c4704d93.png)

Followed the same naming convention as:
- `editorError.border`
- `editorError.foreground`

aeschli commented:
The color needs to be transparent to work with other highlight colors. We should add that information to the description. There's also a boolean parameter on registerColor to signal that.
Example:
https://github.com/microsoft/vscode/blob/07f0bade3702808ff504d4beaf6b0d813dc38c75/src/vs/editor/common/view/editorColorRegistry.ts#L16

What about the other problem types? Warning, info, ...

misolori commented:
@aeschli thanks for the pointer, I'll update this and also add the other types.

misolori commented:
Updated and added `editorWarning.background` and `editorInfo.background`:

![image](https://user-images.githubusercontent.com/35271042/98583704-a9cb6b00-2279-11eb-9d8b-0a57392f7e7f.png)
